### PR TITLE
Update dlp-learn-about-dlp.md

### PR DESCRIPTION
**Pull request by msbemba**
Added permissions and roles needed for DLP

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7497

File: microsoft-365/compliance/dlp-learn-about-dlp.md [hunk 1]
```diff
@@ -31,6 +31,43 @@ In Microsoft 365, you implement data loss prevention by defining and applying DL
 
 Microsoft 365 detects sensitive items by using deep content analysis, not by just a simple text scan. Content is analyzed for primary data matches to keywords, by the evaluation of regular expressions, by internal function validation, and by secondary data matches that are in proximity to the primary data match. Beyond that DLP also uses machine learning algorithms and other methods to detect content that matches your DLP policies.
 
+## Permissions
+
+Members of your compliance team who will create DLP policies need permissions to the Compliance Center. By default, your tenant admin will have access can give compliance officers and other people access. Follow these steps:
+  
+1. Create a group in Microsoft 365 and add compliance officers to it.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-learn-about-dlp.md:38]:
```suggestion
1. Create a group in Microsoft 365 and add compliance officers to it.
```

File: microsoft-365/compliance/dlp-learn-about-dlp.md [hunk 2]
```diff
@@ -31,6 +31,43 @@ In Microsoft 365, you implement data loss prevention by defining and applying DL
...
+    
+2. Create a role group on the **Permissions** page of the Security &amp; Compliance Center. 
+
+3. While creating the role group, use the **Choose Roles** section to add the following role to the role group: **DLP Compliance Management**.
+    
+4. Use the **Choose Members** section to add the Microsoft 365 group you created before to the role group.
+
+Use the **View-Only DLP Compliance Management** role to create role group with view-only privileges to the DLP policies and DLP reports.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-learn-about-dlp.md]:
```suggestion
Use the **View-Only DLP Compliance Management** role to create a role group with view-only privileges to the DLP policies and DLP reports.
```

File: microsoft-365/compliance/dlp-learn-about-dlp.md [hunk 3]
```diff
@@ -31,6 +31,43 @@ In Microsoft 365, you implement data loss prevention by defining and applying DL
...
+
+For more information, see [Give users access to the Office 365 Compliance Center](../security/office-365-security/grant-access-to-the-security-and-compliance-center.md).
+  
+These permissions are required to create and apply a DLP policy not to enforce policies.
+
+### Roles and Role Groups in preview
+
+There are roles and role groups in preview that you can test out to fine tune your access controls.
+
+Here's a list of Microsoft Information Protection (MIP) roles that are in preview. To learn more about them, see [Roles in the Security & Compliance Center](../security/office-365-security/permissions-in-the-security-and-compliance-center.md#roles-in-the-security--compliance-center)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-learn-about-dlp.md]:
```suggestion
Here's a list of Microsoft Information Protection (MIP) roles that are in preview. To learn more about them, see [Roles in the Security & Compliance Center](../security/office-365-security/permissions-in-the-security-and-compliance-center.md#roles-in-the-security--compliance-center).
```

File: microsoft-365/compliance/dlp-learn-about-dlp.md [hunk 4]
```diff
@@ -31,6 +31,43 @@ In Microsoft 365, you implement data loss prevention by defining and applying DL
...
+
+- Information Protection Admin
+- Information Protection Analyst
+- Information Protection Investigator
+- Information Protection Reader
+
+Here's a list of MIP role groups that are in preview. To learn more, see [Role groups in the Security & Compliance Center](../security/office-365-security/permissions-in-the-security-and-compliance-center.md#role-groups-in-the-security--compliance-center)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-learn-about-dlp.md]:
```suggestion
Here's a list of MIP role groups that are in preview. To learn more, see [Role groups in the Security & Compliance Center](../security/office-365-security/permissions-in-the-security-and-compliance-center.md#role-groups-in-the-security--compliance-center).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

File: microsoft-365/compliance/dlp-learn-about-dlp.md [hunk 5]
```diff
@@ -31,6 +31,43 @@ In Microsoft 365, you implement data loss prevention by defining and applying DL
 
 Microsoft 365 detects sensitive items by using deep content analysis, not by just a simple text scan. Content is analyzed for primary data matches to keywords, by the evaluation of regular expressions, by internal function validation, and by secondary data matches that are in proximity to the primary data match. Beyond that DLP also uses machine learning algorithms and other methods to detect content that matches your DLP policies.
 
+## Permissions
+
+Members of your compliance team who will create DLP policies need permissions to the Compliance Center. By default, your tenant admin will have access can give compliance officers and other people access. Follow these steps:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-learn-about-dlp.md]:
```suggestion
Members of your compliance team who will create DLP policies need permissions to the Compliance Center. By default, your tenant admin will have access and can give compliance officers and other people access. Follow these steps:
```

File: microsoft-365/compliance/dlp-learn-about-dlp.md [hunk 6]
```diff
@@ -31,6 +31,43 @@ In Microsoft 365, you implement data loss prevention by defining and applying DL
...
+
+3. While creating the role group, use the **Choose Roles** section to add the following role to the role group: **DLP Compliance Management**.
+    
+4. Use the **Choose Members** section to add the Microsoft 365 group you created before to the role group.
+
+Use the **View-Only DLP Compliance Management** role to create a role group with view-only privileges to the DLP policies and DLP reports.
+
+For more information, see [Give users access to the Office 365 Compliance Center](../security/office-365-security/grant-access-to-the-security-and-compliance-center.md).
+  
+These permissions are required to create and apply a DLP policy not to enforce policies.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-learn-about-dlp.md]:
```suggestion
These permissions are required to create and apply a DLP policy, not to enforce policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, two more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Kindly merge this PR. Thanks

**Comment by chrfox** [comment]
This content is already presented in https://docs.microsoft.com/en-us/microsoft-365/compliance/create-test-tune-dlp-policy?view=o365-worldwide, you've just cut and pasted it into the purely conceptual Learn about article where it doesn't belong

**Pull request closed without merging**